docs/new-guide-how-to-launch-a-new-terraform-building-block.md

```diff
@@ -60,11 +60,31 @@ A Building Block is created to operate on one or several Platforms. This selecti
 
 How often can the same Building Block be allocated to a particular project? Choose "One" for a single assignment, or "Multiple" for more than one.
 
-#### Terraform vs manual
+#### meshStack Built-in HTTP Backend
 
-The building block definition has two implementation types: Terraform and manual. Using Terraform, the building block can automatically provision the infrastructure resources. This gives engineers and DevOps the ability to use the full power of IaC (Infrastructure as Code) to create self-service products available in the Marketplace.
+meshStack can automatically manage Terraform/OpenTofu state through its built-in HTTP backend.
+When enabled in a building block definition, meshStack injects the backend configuration
```

Comment:
d: Should you also mention here, that you have to create an API Key and provide it as env vars to the BB